### PR TITLE
Bug/446

### PR DESCRIPTION
**Pull request by OlegGordiichuk**
Add scroll view

### CODE REVIEW

File: Trust/Tokens/ViewControllers/NFTokenViewController.swift [hunk 1]
```diff
@@ -52,6 +66,27 @@ class NFTokenViewController: UIViewController {
         externalButton.setTitle(viewModel.externalButtonTitle, for: .normal)
         externalButton.addTarget(self, action: #selector(externalTap), for: .touchUpInside)
 
+        let externalButton1 = Button(size: .normal, style: .border)
```
Comment (vikmeup) [inline Trust/Tokens/ViewControllers/NFTokenViewController.swift]: Can you add space between each button

Comment (vikmeup) [reply Trust/Tokens/ViewControllers/NFTokenViewController.swift]: Can you remove test buttons?

File: Trust/Tokens/ViewControllers/NFTokenViewController.swift [hunk 2]
```diff
@@ -11,7 +11,23 @@ protocol NFTokenViewControllerDelegate: class {
 
 class NFTokenViewController: UIViewController {
 
-    let stackView = UIStackView()
+    lazy var scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        scrollView.showsHorizontalScrollIndicator = true
```
Comment (vikmeup) [inline Trust/Tokens/ViewControllers/NFTokenViewController.swift]: We only need vertical scrolling enabled

File: Trust/Tokens/ViewControllers/NFTokenViewController.swift (hunk 1 above)
Comment (OlegGordiichuk) [reply Trust/Tokens/ViewControllers/NFTokenViewController.swift]: Done

**Pull request merged**